test/extensions/filters/http/fault/fault_filter_integration_test.cc

```diff
@@ -0,0 +1,137 @@
+#include "test/integration/http_protocol_integration.h"
+
+#include "gtest/gtest.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace HttpFilters {
+namespace Fault {
+namespace {
+
+class FaultIntegrationTest : public Event::TestUsingSimulatedTime,
+                             public HttpProtocolIntegrationTest {
+public:
+  void initializeFilter(const std::string& filter_config) {
```

Comment:
optional: you could put these both in SetUp() and not call it per-test.

Reply:
I use a different config in one of the tests?